2023-10-27 00:03:17,791 INFO    MainThread:23724 [wandb_setup.py:_flush():76] Current SDK version is 0.15.3
2023-10-27 00:03:17,791 INFO    MainThread:23724 [wandb_setup.py:_flush():76] Configure stats pid to 23724
2023-10-27 00:03:17,791 INFO    MainThread:23724 [wandb_setup.py:_flush():76] Loading settings from C:\Users\tobia\.config\wandb\settings
2023-10-27 00:03:17,792 INFO    MainThread:23724 [wandb_setup.py:_flush():76] Loading settings from C:\Users\tobia\Documents\GitHub\RNN_Tobia_Simmler\wandb\settings
2023-10-27 00:03:17,792 INFO    MainThread:23724 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-10-27 00:03:17,792 INFO    MainThread:23724 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-10-27 00:03:17,792 INFO    MainThread:23724 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program': 'C:\\Users\\tobia\\Documents\\GitHub\\RNN_Tobia_Simmler\\main.py'}
2023-10-27 00:03:17,792 INFO    MainThread:23724 [wandb_init.py:_log_setup():507] Logging user logs to C:\Users\tobia\Documents\GitHub\RNN_Tobia_Simmler\wandb\run-20231027_000317-eq3knvuo\logs\debug.log
2023-10-27 00:03:17,792 INFO    MainThread:23724 [wandb_init.py:_log_setup():508] Logging internal logs to C:\Users\tobia\Documents\GitHub\RNN_Tobia_Simmler\wandb\run-20231027_000317-eq3knvuo\logs\debug-internal.log
2023-10-27 00:03:17,792 INFO    MainThread:23724 [wandb_init.py:init():547] calling init triggers
2023-10-27 00:03:17,792 INFO    MainThread:23724 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {'dataset_name': 'ctxt', 'n_epochs': 500, 'batch_size': 16, 'total_seq_length': 1400, 'seq_length': 1400, 'training_help': 5, 'hidden_dims': 100, 'n_trials': 5, 'with_inputnoise': 1, 'noise_level': 2, 'coherency_intervals': 'uniform', 'optimizer_name': 'adam', 'lr': 0.001, 'betas': [0.9, 0.999], 'eps': 1e-07, 'weight_decay': 0.0001, 'amsgrad': False, 'momentum': 0.9, 'apply_gradient_clipping': 0}
2023-10-27 00:03:17,792 INFO    MainThread:23724 [wandb_init.py:init():596] starting backend
2023-10-27 00:03:17,792 INFO    MainThread:23724 [wandb_init.py:init():600] setting up manager
2023-10-27 00:03:17,793 INFO    MainThread:23724 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-10-27 00:03:17,795 INFO    MainThread:23724 [wandb_init.py:init():606] backend started and connected
2023-10-27 00:03:17,796 INFO    MainThread:23724 [wandb_init.py:init():700] updated telemetry
2023-10-27 00:03:18,029 ERROR   MainThread:23724 [wandb_init.py:init():1178] error
Traceback (most recent call last):
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\site-packages\wandb\sdk\wandb_init.py", line 1150, in init
    run = wi.init()
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\site-packages\wandb\sdk\wandb_init.py", line 718, in init
    run_proto = backend.interface._make_run(run)
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\site-packages\wandb\sdk\interface\interface.py", line 173, in _make_run
    run._make_proto_run(proto_run)
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\site-packages\wandb\sdk\wandb_run.py", line 773, in _make_proto_run
    run.display_name = self._name
TypeError: 0 has type int, but expected one of: bytes, unicode
